index.d.ts

```diff
@@ -115,25 +118,14 @@ export type Options = {
 	/**
 	Color of the background.
 	*/
-	readonly backgroundColor?: LiteralUnion<
-	| 'black'
-	| 'red'
-	| 'green'
-	| 'yellow'
-	| 'blue'
-	| 'magenta'
-	| 'cyan'
-	| 'white'
-	| 'blackBright'
-	| 'redBright'
-	| 'greenBright'
-	| 'yellowBright'
-	| 'blueBright'
-	| 'magentaBright'
-	| 'cyanBright'
-	| 'whiteBright',
-	string
-	>;
+	readonly backgroundColor?: Color;
+
+	/**
+	Color of the background of the border.
+
+	Defaults to `backgroundColor`. Use `'none'` for no color.
+	*/
+	readonly borderBackgroundColor?: Color;
```

Comment:
```suggestion
	readonly borderBackgroundColor?: Color | undefined;
```

This would be better than `'none'`.

Reply:
That wouldn't work because now there is no way to differentiate between "unspecified" and "explicitly no color". Maybe should we revert this back ?

Reply:
But it should not be in the general `Color` type. Only this property needs it. And what do you think of instead having a `'inherit'` case, which it would default to?

So:

- `{borderBackgroundColor: undefined}` => no color
- `{borderBackgroundColor: 'inherit'}` => inherit color
- `{}` => inherit color